docs/intro/features.rst

```diff
@@ -44,12 +42,15 @@ Several pre-trained models are available and presented in Table below.
 +-------------------+--------------------------------------------------------------------------------------------------------------+------------------+------+----------------+---------------+
 |`Twitter mokoron`_ | :config:`Twitter on RuWiki+Lenta embeddings with preprocessing <classifiers/sentiment_twitter_preproc.json>` | Sentiment        | Ru   | 0.7944         | 0.7879        |
 +-------------------+--------------------------------------------------------------------------------------------------------------+------------------+------+----------------+---------------+
+|`RuSentiment`_     | :config:`RuSentiment on RuWiki+Lenta embeddings <classifiers/rusentiment_cnn.json>`                          | Sentiment        | Ru   | 0.6822         | 0.7095        |
```

Comment:
I see F1, precision and recall on the project site... how should I use these values of accuracy for comparison?
Also, what about embeddings pre-trained on Vkontakte corpus?

Reply:
you can just evaluate this model by yourself.

Reply:
the task was not to reproduce presented models, just to train some model in our library